src/webui/www/private/views/preferences.html

```diff
@@ -7,6 +7,30 @@
         </select>
     </fieldset>
 
+    <fieldset class="settings">
+        <legend>QBT_TR(Action on double-click)QBT_TR[CONTEXT=OptionsDialog]</legend>
+        <table>
+            <tr>
+                <td><label for="dblclick_download_select">QBT_TR(Downloading torrents:)QBT_TR[CONTEXT=OptionsDialog]</label></td>
+                <td>
+                    <select id="dblclick_download_select">
+                        <option value="0">QBT_TR(No action)QBT_TR[CONTEXT=OptionsDialog]</option>
+                        <option value="1" selected>QBT_TR(Start / Stop Torrent)QBT_TR[CONTEXT=OptionsDialog]</option>
```

Comment:
Why not use the same order as in GUI?
The same in `dblclick_complete_select`.

Reply:
This order looks more logical to me. Representing state as `0` - off, `1` - on.
Also future proof in case of adding extra actions.

Reply:
Ok, it's easy to reorder the list visually.